validator/src/main/resources/expected-data-template/container-insight/eks/infrastructure/ClusterNamespace.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "structured log schema",
+    "description": "json schema for the cloudwatch agent k8s structured log",
+    "type": "object",
+    "properties": {
+      "ClusterName":{},
+      "Type":{},
+      "NodeName": {},
+      "Sources":{},
+      "Timestamp":{},
+      "Version":{},
+      "Namespace":{},
+      "namespace_number_of_running_pods":{},
+      "_aws": {},
```

Comment:
Better to add more validation under `_aws`, e.g.
https://github.com/aws-observability/aws-otel-test-framework/blob/911979fedeb9a520e2b90906645ea6835b39842e/validator/src/main/resources/expected-data-template/container-insight/eks/prometheus/appMesh.json#L10-L19

Reply:
Right! I will add them.

Reply:
Actually since we are checking metrics after structured log, we don't really need to check those emf related fields 🐶